lineQueueClient/src/styles/globals.style.js

```diff
@@ -0,0 +1,18 @@
+import { StyleSheet, Dimensions } from 'react-native';
+
+const globals = StyleSheet.create({
+  sceneContainer: {
+    display:'flex',
+    width: Dimensions.get('window').width,
```

Comment:
To request window width and height do:
```js
let {height, width} = Dimensions.get('window')
```
and then:
```json
{
  width: width,
  height: height
}
```

Reply:
fixed